test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Feeds/InstalledAndTransitivePackageFeedTests.cs

```diff
@@ -232,8 +232,13 @@ public async Task SearchAsync_WhenMultitargetingAndEmptySearch_ReturnsOnePackage
                 e => Assert.Equal(new PackageIdentity("packageA", new NuGetVersion("1.0.0")), e.Identity),
                 e => Assert.Equal(new PackageIdentity("packageB", new NuGetVersion("2.0.0")), e.Identity),
                 e => Assert.Equal(new PackageIdentity("packageC", new NuGetVersion("3.0.0")), e.Identity),
+                // Returns all the installed packages that are not the latest version
+                e => Assert.Equal(new PackageIdentity("packageB", new NuGetVersion("1.0.0")), e.Identity),
+                e => Assert.Equal(new PackageIdentity("packageC", new NuGetVersion("2.0.0")), e.Identity),
                 // Returns latest transitive package version
-                e => Assert.Equal(new PackageIdentity("transitivePackageC", new NuGetVersion("0.0.2")), e.Identity));
+                e => Assert.Equal(new PackageIdentity("transitivePackageC", new NuGetVersion("0.0.2")), e.Identity),
+                // Returns all the transtive packages that are not the latest verion
```

Comment:
Could the comment be expanded to explain why this assertion is made?

```suggestion
                // Returns all the transitive packages that are not the latest version because.....
```

Reply:
So this PackageCollection is sorted, first we display the Installed Packages and then the TransitivePackages, and in each "group" they are sorted by latest. 

In this test I'm trying to test that they are sorted as expected

Reply:
If they are sorted, why is transitivePackageC 0.0.2 asserted before version 0.0.1 ?

Are you planning to fix the typos I suggested?